v1.0/inference.py

```diff
@@ -0,0 +1,34 @@
+from fastai2.vision.all import *
+
+# REPLACE THIS MODEL WITH MOST UP TO DATE VERSION
+MODEL = 'rn50-448px-v0.9-export.pkl'
+
+
+def get_lbl(fname):
```

Comment:
please mention that the model when loaded absolutely needs this function to exist

Reply:
thanks

Reply:
ahm, is that 0.8 or 0.9 now?